### PR TITLE
Switch to GH Actions

### PR DESCRIPTION
**Pull request by le0m**
This PR switches CI from Travis.org to GH Actions

### CODE REVIEW

**Review by fquffio — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    if: "!contains(github.event.commits[0].message, '[skip ci]') && !contains(github.event.commits[0].message, '[ci skip]')"
+    needs: cancel-previous-runs
+    strategy:
+      matrix:
+        version: [ '5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0' ]
+        flavor: [ '', '-apache', '-fpm' ]
+        include:
+          - version: 'latest'
+            flavor: ''
+    steps:
```
Comment (ad-m) [inline .github/workflows/main.yml:33]: I suggest define that globally:

```suggestion
    env:
        VERSION: "${{ matrix.version }}${{ matrix.flavor }}"
    steps:
```

Then remove from each step individually. See example at https://github.com/hyperonecom/h1-images-recommended/blob/master/.github/workflows/image.yml#L38 .

**Comment by ad-m** [comment]
Error `Peer fingerprint did not match` according https://twitter.com/akrabat/status/1031568085810065408 is related to external API unreliability, but...

We have similiar error on TravisCI ( https://travis-ci.org/github/chialab/docker-php/builds/747420146 ) and https://programmersought.com/article/82073298856/ suggestion PHP 5.5 incompatibility. We attempt to use compose version `2.0.9` which is latest version ( https://github.com/composer/composer/releases ). That error exist in code flow specific to PHP 5.5 ( https://github.com/composer/composer/blob/b7d770659b4e3ef21423bd67ade935572913a4c1/src/Composer/Util/RemoteFilesystem.php#L320 ).

**Comment by le0m** [comment]
We're installing both Composer v1 and Composer v2, but the error happens when using Composer v1 to install the package `hirak/prestissimo`.
You are right that it's specific to PHP < 5.6, but I couldn't reproduce locally when building Docker image with PHP 5.4/5.5.
I'm still working on this...thanks for your help!

**Comment by pulponair** [comment]
Any news on this?

**Comment by le0m** [comment]
Sorry for the delays, in the end I disabled `hirak/prestissimo` in PHP 5.4 and 5.5 images

**Pull request merged**